CONTRIBUTING.md

```diff
@@ -129,12 +129,15 @@ hivemind
 
 ### Resetting the database
 
-In order to start over, delete the database and recreate it, then restore it from a dump, and (just in case the dump is behind the code) run migrations:
+In order to start over, delete the database and recreate it, then restore it from staging, and (just in case the dump is behind the code) run migrations.
+
+> [!NOTE]
+> For now this requires SSH access to the staging environment. In the future you will be able to download a publicly available dump.
 
 ```bash
 dropdb nix-security-tracker
 sudo -u postgres createdb -O nix-security-tracker nix-security-tracker
-pg_restore -O -d nix-security-tracker -v < dump
+ssh sectracker.nixpkgs.lahfa.xyz "sudo -u postgres pg_dump --create web-security-tracker | zstd" | zstdcat | sed 's|web-security-tracker|nix-security-tracker|g' | pv | psql
```

Comment:
Please make that an *addition* so we don't lose track of the direct-from-dump workflow.

Reply:
done